src/Shared/SignalVersionHelper.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Reflection;
+
+namespace OpenTelemetry.Instrumentation;
+
+internal static class SignalVersionHelper
+{
+    public static string GetVersion<T>()
```

Comment:
Update the function name to express the intention here ("get version" - get what version?). Add comments to describe the logic here.

Reply:
Class was renamed to  InstrumentationScopeHelper in 33d5521a73e881ac59d4bf1213765270ec2422ff.

`InstrumentationScopeHelper.GetVersion<Type>()` look pretty self descriptive. Implementation details added in 4294446cff57b068dc3ad85fd66f993e34bca490.

Please let me know if you are fine with this.